shared/README.md

```diff
@@ -0,0 +1,14 @@
+﻿# dd-trace-dotnet/shared
+
+This repo ([dd-trace-dotnet](https://github.com/DataDog/dd-trace-dotnet)) contains the sources for the client-side components of the Datadog product suite for Application Telemetry Collection and Application Performance Monitoring for .NET Applications.
+
+This directory (`/shared`) contains code code which is shared between between the different products of that suite (.NET Tracer, .NET Profiler, etc...).
+
+### Copyright
+
+Copyright (c) 2017-2021 Datadog
+[https://www.datadoghq.com](https://www.datadoghq.com/)
+
+### License
+
+See [license information](./LICENSE).
```

Comment:
```suggestion
See [license information](../LICENSE).
```

Reply:
Addressed in https://github.com/DataDog/dd-trace-dotnet/commit/8b9642541f0ad21873f3bdd9325b6601a10bb8c9